apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx

```diff
@@ -75,83 +81,86 @@ export default function SelectFeesStrategy({
   const closeNetworkFeeHelpModal = () => setNetworkFeeHelpOpened(false);
 
   const onPressStrategySelect = useCallback(
-    (item: FeeStrategy) => {
+    (item: SelectFeeStrategy) => {
```

Comment:
You can add second parameter userGasLimit here, so you dont' need to create a new type "SelectFeeStrategy". It would be more logic in my opinion.

Reply:
I didn't create type for this function I created it to remove all the `as` statements and object intersections in the code